src/apps/announcements/models/announcement.py

```diff
@@ -0,0 +1,58 @@
+
+from django.db.models import (
+    Model,
+    BooleanField,
+    CharField,
+    DateTimeField,
+    IntegerField,
+    TextField,
+    BigAutoField,
+)
+from django.utils.translation import gettext_lazy as _
+
+class Announcement(Model):
+    """
+    An announcement is a block of html text for the front page display.
+    """
+
+    class Meta:
+        verbose_name = _("Announcement")
+        verbose_name_plural = _("Announcements")
+        ordering = ["-display_order"]
+
+    id = BigAutoField(primary_key=True)
+    created_at = DateTimeField(_("creation date"), auto_now_add=True, db_index=True)
+    updated_at = DateTimeField(_("updated date"), auto_now=True, db_index=True)
+    title = CharField(
+        _("title"),
+        null=False,
+        blank=False,
+        max_length=240,
+        help_text=_("Title of the announcement section."),
+        db_index=True,
+        unique=True,
+    )
+    contents = TextField(
```

Comment:
Did you prefer raw html as something like markdown ?

Easier to code, but maybe a bit less clear than md to write.
Plus security breach vector if at some points people other than me or you can write annoncement (stored XSS).

Reply:
Thanks, made it markdown. But I still only intend admins to write. :)